### ISSUE TITLE
Receiving type error for Transport::convertEmailsArray

### ISSUE THREAD

**Issue by cgsmith** [wildbit/swiftmailer-postmark#17]
I received this error when only setting a BCC message.
```
Uncaught TypeError: Argument 1 passed to Postmark\Transport::convertEmailsArray() must be of the type array, null given, called in Transport.php on line 138 and defined in Transport.php:82
Stack trace:
#0 Transport.php(138): Postmark\Transport->convertEmailsArray(NULL)
#1 Transport.php(118): Postmark\Transport->processRecipients(Array, Object(Swift_Message))
#2 Transport.php(70): Postmark\Transport->getMessagePayload(Object(Swift_Message))
#3 Transport->send(Ob
```

After inspecting it though it looks like it is set.

Swiftmailer 5.2

![image](https://user-images.githubusercontent.com/570018/53598382-34ce0580-3b6a-11e9-82ab-6eaaf2661516.png)

**Comment by atheken** [comment]
Thanks Chris, we’ll have a look and get it fixed in the next 24 hours.

//Andrew

**Comment by atheken** [comment]
Thanks again @cgsmith -- merged and published as 3.0.2 to packagist. If you bump into anything else, please let us know.
